packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -378,6 +378,9 @@ class _CupertinoDatePickerDateTimeState extends State<CupertinoDatePicker> {
 
   // The controller of the AM/PM column.
   FixedExtentScrollController amPmController;
+  FixedExtentScrollController hourController;
```

Comment:
I'm assuming changes in this class is unrelated right? It's ok but we also need to make sure this change into reading  values from a CupertinoDatePickerController instead of just `selectedDayFromInitial` etc in the future.

Reply:
Ok, yah I changed that initially to make its class work with the controller, but I felt that it was a good to change to make anyway.

Reply:
We can keep these